tensorflow_lite_support/python/task/processor/proto/BUILD

```diff
@@ -22,3 +22,24 @@ py_library(
     srcs = ["bounding_box_pb2.py"],
     deps = ["//tensorflow_lite_support/cc/task/vision/proto:bounding_box_py_pb2"],
 )
+
+py_library(
```

Comment:
For consistency with the latest changes on ImageEmbedder: this one (image_classifier_options) should live under python/task/vision/pybinds. The rationale between this is that this proto is never exposed to the user directly.

Reply:
@mbrenon I'll be closing this PR since there are major changes needed. I thought of raising this larger PR to get some early feedback FYI